src/components/login/Login.js

```diff
@@ -22,11 +21,9 @@ class Login extends Component {
         })
     }
 
-    handleDeny = e => {
-        e.preventDefault();
-        if (this.props.account.url.failure_url) {
-            window.location.href = this.props.account.url.failure_url
-        }
+    handleDeny = () => {
+        const failureUrl = this.props.account.url.failure_url;
+        failureUrl ? window.location.href = failureUrl : this.props.redirectToApp();
```

Comment:
nit: this is very confusing to read as expression (as result is ignored anyway), why not just use `if` statement?

Reply:
Funny that you say that because I was thinking about it for a second and thought that it was nice to read. I see your point though. Fixed.

Reply:
It's a bit shorter, but it can be hard to read, because there are side-effects happening in what is normally an expression by syntax. `=` in expression in general adds additional level of confusion